modules/extension/wmts/src/main/java/org/geotools/ows/wmts/map/WMTSCoverageReader.java

```diff
@@ -664,48 +677,31 @@ private GridCoverage2D getMapReproject(
     CoordinateReferenceSystem getBestSourceCRS(
             CoordinateReferenceSystem targetCRS, WMTSReadParameters readParameters)
             throws IOException {
-        try {
-            CoordinateReferenceSystem result = null;
-            String switched = switchDefinition(targetCRS);
```

Comment:
Why also remove support for switched axis match? This is also not random.

Reply:
Nothing else than getRandomCrs are removed. The display og the change is somewhat misleading because I moved a try-catch. With this change I mean to respect the user's crs setting. That was my problem with the getRandomCrs.

Reply:
You are right. Can you address the other feedback?